test/type/opaque_return_type_obc_protocol.swift

```diff
@@ -0,0 +1,15 @@
+// RUN: %target-swift-frontend -disable-availability-checking -emit-ir -verify %s
+//
+// Ensures this does not cause a crash, as @objc protocols are a special case
+// because they do not require a witness table
+// dar://problem/59740179
```

Comment:
```suggestion
// rdar://problem/59740179
```

Also, I think you also need to add `// REQUIRES: objc_interop`